test/view_models/workarea/storefront/content_search_autocomplete_view_model_test.rb

```diff
@@ -0,0 +1,45 @@
+require 'test_helper'
+
+module Workarea
+  if Plugin.installed?(:search_autocomplete)
+    module Storefront
+      class SearchAutocompleteViewModelTest < Workarea::TestCase
+        include TestCase::SearchIndexing
+
+        def test_content
+          query = QueryString.new('te').pretty
+          search = Storefront::SearchAutocompleteViewModel.wrap(query)
+
+          Sidekiq::Callbacks.enable(*index_jobs) do
+            create_page(name: 'Content Test One')
+            create_page(name: 'Content Test Two')
+            create_page(name: 'Content Test Three')
+            create_product(name: 'Test One')
+            create_product(name: 'Test Two')
+            create_search_by_week(
+              query_string: 'test one',
+              searches: 5,
+              total_results: 5
+            )
+            create_search_by_week(
+              query_string: 'test two',
+              searches: 10,
+              total_results: 5
+            )
+          end
+
+          refute_empty(search.content_results)
+          assert_equal(3, search.content_results.count)
+        end
+
+        private
+
+        def index_jobs
```

Comment:
I made this a private method in case someone adds a `Sidekiq::Callback` for `Catalog::Product` or `Content::Page`, so they wouldn't have to override the actual test. Didn't want to just `.enable` all jobs for the same reason..

Reply:
Not sure what you mean, what's the harm in just enabling workers?

Reply:
yeah you're probably right i was overthinking it. went back to just enabling all workers for those lines.